sysdeps/linux/x86_64/generic.c

```diff
@@ -0,0 +1,14 @@
+#include <stdarg.h>
+#include <sys/syscall.h>
+
+static void
+syscall1(int nbr, unsigned long arg)
```

Comment:
I feel like putting the syscall function somewhere in the main libc should be ok :thinking: 
What do you think ?

Reply:
no, it wouldn't be standard compliant